components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/RejectedGeneListController.java

```diff
@@ -142,8 +144,12 @@ public void writeJSON(Patient patient, JSONObject json, Collection<String> selec
                 json.put(getJsonPropertyName(), new JSONArray());
                 container = json.getJSONArray(getJsonPropertyName());
             }
+
+            if (item != null && "".equals(item.get(COMMENTS_KEY))) {
```

Comment:
Use this:

    if (item != null && StringUtils.isBlank(item.get(COMMENTS_KEY))) {